src/inmanta_ui/config.py

```diff
@@ -17,19 +17,26 @@
 """
 from inmanta.config import Option, is_bool, is_str
 
-web_console_enabled = Option("web-console", "enabled", True, "Should the server should host the web-console or not", is_bool)
+web_console_enabled = Option("web-ui", "enabled", True, "Whether the server should host the web-console or not", is_bool)
```

Comment:
The idea behind the `web-ui` section is that it's generic and allows for more web-interfaces to be added in the future. From that perspective we probably don't want an `enabled` flag, but probably a `web_console_enabled` flag or something similar. @bartv Do you have a strong opinion here?

Reply:
I would say console_enabled then.

We can even say: do we need an option to disable it?